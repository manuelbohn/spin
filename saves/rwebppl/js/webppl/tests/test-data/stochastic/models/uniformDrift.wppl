var model = function() {
  var x = uniformDrift({a: -3, b: 6, width: 0.2});
  condition(x >= 0);

  return x;
};
