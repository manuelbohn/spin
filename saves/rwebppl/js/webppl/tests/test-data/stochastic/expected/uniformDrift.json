{
  "mean": 3.0,
  "std": 1.732050808
}
